openslides/core/templates/base.html

```diff
@@ -89,13 +89,14 @@
     <div class="container-fluid" id="container">
     <div class="row">
 
-        <!-- Sidebar navigation (main menu) -->
+        <!-- Sidebar navigation (main menu)
+             TODO: this menu is a temporary solution only! We will use angular base template
+             for building main menu, see issue #1469 -->
         <div class="col-md-2 leftmenu lefticon">
             <ul>
                 {% for entry in main_menu_entries %}
-                    <li{% if entry.is_active %} class="active"{% endif %}>
-                        <a href="{{ entry.get_url }}" class="tooltip-right">
-                        <!--TODO-->
+                    <li>
+                        <a href="{{ entry.pattern_name }}" class="tooltip-right">
```

Comment:
this base template is not an angular template and will be removed is the template work finished. My changes are only for getting faster the specific app url for testing. So I don't want to do more effort in this depricated template.